src/main/java/com/example/schooloperationsystem/service/params/CreateSchoolClassParams.java

```diff
@@ -8,7 +8,7 @@
  */
 @Getter
 @Setter
-public class SchoolCreateClassParams {
+public class CreateSchoolClassParams {
```

Comment:
please create all arguments constructor and in constructor check that fields are not null, if they are, please throw illegal argument exception